src/app/Components/MainDashbordComponents/footer/footer.component.html

```diff
@@ -105,21 +105,21 @@ <h5>
     </div>
 
     <div class="col md-4 lg-4">
-        <h5><span style="color: #29416F; font-family: outfit;">Contact Info</span></h5>
+        <h5><span style="color: var(--secondary-color); font-family: outfit;"> Contact Info</span></h5>
```

Comment:
Please remove inline style, instead use same class for primary-color class and template-font class from common